deploy/base/kustomization.yaml

```diff
@@ -33,7 +33,8 @@ configMapGenerator:
       - enableModel=false
   - name: nl-config
     files:
-      - model.yaml
+      - embeddings.yaml
+      - tunedmodel.yaml
```

Comment:
This file is not used so no need to update here

Reply:
Got it.